fe/fe-core/src/main/java/org/apache/doris/nereids/parser/LogicalPlanBuilder.java

```diff
@@ -531,7 +531,8 @@ public LogicalPlan visitInsertTable(InsertTableContext ctx) {
         if (isOverwrite) {
             command = new InsertOverwriteTableCommand(sink, labelName);
         } else {
-            if (ConnectContext.get() != null && ConnectContext.get().isTxnModel()) {
+            if (ConnectContext.get() != null && ConnectContext.get().isTxnModel()
+                    && sink.child() instanceof LogicalInlineTable) {
```

Comment:
In legacy planner, we support `insert into t select  1`. So if we only support `LogicalInlineTable` here means we have diff with legacy planner

Reply:
The nereids does not support `insert into t select  1`? It fallback to legacy